### PR TITLE
Add back events with no matches at the bottom

### PR DESCRIPTION
**Pull request by dash102**
This PR:
- Adds back events that are not in directMatchEvents or similarMatchEvents

### CODE REVIEW

File: project/src/main/java/com/google/sps/utilities/EventRanker.java [hunk 1]
```diff
@@ -46,16 +48,31 @@
...
-      eventScoreMap.put(eventRelevancyPair.getValue0(), new Double(eventRelevancyPair.getValue1()));
-    }
-    for (Pair<Event, Integer> eventRelevancyPair : similarEventPairs) {
-      eventScoreMap.put(eventRelevancyPair.getValue0(), SIMILAR_WEIGHT * eventRelevancyPair.getValue1());
+    Set<Event> directEvents = processEventMatches(eventScoreMap, allEvents.getValue0(), DIRECT_WEIGHT);
+    Set<Event> similarEvents = processEventMatches(eventScoreMap, allEvents.getValue1(), SIMILAR_WEIGHT);
+    for (Event event : getNoMatchEvents(events, directEvents, similarEvents)) {
+      eventScoreMap.put(event, NO_MATCH_WEIGHT);
     }
+    System.out.println(eventScoreMap.size());
```
Comment (hlz03) [inline project/src/main/java/com/google/sps/utilities/EventRanker.java:57]: Remove this?

**Comment by nikhitashanker** [comment]
LGTM after removing println.

File: project/src/main/java/com/google/sps/utilities/EventRanker.java (hunk 1 above)
Comment (dash102) [reply project/src/main/java/com/google/sps/utilities/EventRanker.java:57]: Oops, thanks for catching - done

File: project/src/main/java/com/google/sps/utilities/EventRanker.java [hunk 2]
```diff
@@ -16,7 +16,9 @@
 import org.javatuples.Pair;
 
 public class EventRanker {
-  private static Double SIMILAR_WEIGHT = 0.7;
+  private static final Double DIRECT_WEIGHT = 1.0;
+  private static final Double SIMILAR_WEIGHT = 0.7;
+  private static final Double NO_MATCH_WEIGHT = 0.0;
 
   /**
    * Sorts events based on each event's score. Breaks ties with the event's date; the more recent
```
[the remark this answers is not in the document]
Comment (dash102) [reply project/src/main/java/com/google/sps/utilities/EventRanker.java:24]: Ah thanks for catching, done

**Pull request merged**